java/src/jmri/jmrit/dispatcher/ActivateTrainFrame.java

```diff
@@ -1884,6 +1910,31 @@ private void setAllocateMethodButtons(int value) {
         }
     }
 
+    /*
+     * ComboBox item.
+     */
+    protected static class TrainDetectionItem {
```

Comment:
No need to change, but this class is not needed. Look at the enum `ActionAudio.Operation`:
https://github.com/JMRI/JMRI/blob/master/java/src/jmri/jmrit/logixng/actions/ActionAudio.java#L172

I have a private constructor in the enum together with the private field _text. Each enum item has the bundle for the enum and the enum class has the toString() method that returns the bundle text.

This way, I can use the enum directly in the JComboBox and doesn't need to have any extra class to handle the bundles.